analyzer/src/funTest/assets/projects/synthetic/npm/package-lock/package.json

```diff
@@ -7,6 +7,7 @@
     "url": "https://github.com/oss-review-toolkit/ort.git"
   },
   "license": "Apache-2.0",
+  "author": "The Author <the.author@copy-right.com> (https://www.the.author.org/index.html)",
```

Comment:
If you using fake domain please use example.com or oss-review-toolkit.org. Who know what someone may host on copy-right.com or author.org

Reply:
there are explicit tlds for this use case, e.g. `.test`, `.example` and `.invalid`. See https://tools.ietf.org/html/rfc2606

Reply:
Changed mail addresses and URLs in two package.json files.